src/utils.ts

```diff
@@ -35,3 +22,7 @@ export function readableTime(time: number, granularity: 's' | 'ms', decimalPlace
   const minutes = Math.floor((time % 3_600_000) / 60_000)
   return `${hours}h ${minutes}m`
 }
+
+function truncate(value: number, decimals = 2) {
```

Comment:
are you fixing something specific here? I purposely avoided using `.toFixed` because it's notoriously bad for rounding.

Here's just one blog about it: https://www.sitepoint.com/number-tofixed-rounding-errors-broken-but-fixable/

You can find several stack overflows talking about the same issue

Reply:
Looks like I forgot to push my last commit, sorry about that.
I am only using toFixed to get the right number of trailing 0s but I could do that manually fairly easily.
My testing showed toFixed was faster than my naive approach though.